devtools/sendit/readme_test.go

```diff
@@ -51,7 +51,11 @@ func TestReadme(t *testing.T) {
 	r, w := io.Pipe()
 	cmd.Stderr = io.MultiWriter(w, os.Stdout)
 	require.NoError(t, cmd.Start())
-	defer cmd.Process.Kill()
+	defer func() {
+		if err := cmd.Process.Kill(); err != nil {
+			t.Error(err)
+		}
+	}()
```

Comment:
This loses reference to `cmd` and defeats the purpose of using defer.

If we can't clean up the process we start, the test itself is invalid, rather than the testable code failing; this, too, should probably be a `panic`. We should also ignore the case that the returned error is `ErrProcessDone`

```suggestion
	defer mustExit(cmd.Process.Kill)
```

Reply:
I tested out your suggestion, and `mustExit` doesn't exist.  Would you rather I make a shim function of `mustExit` and check if we need to panic inside of that, or just pass `cmd` into this anonymous function and check for panic in there?

Reply:
Yes, sorry for the confusion, intended it as pseudo code.

I was thinking something like this:
```go
func mustExit(fn func() error) {
	err := fn()
	if err != nil && !errors.Is(err, os.ErrProcessDone) {
		panic(err)
	}
}
```

Though your suggestion of passing `cmd` instead of the `cmd.Process.Kill` method may read cleaner, I like that.

Reply:
I certainly don't mind doing a shim function like you've shown above.  When looking at the bigger picture, my only thought on it would be we probably want to be consistent across the app with handling defers that require some kind of error checking (even if `_ = something()`).  I'll wait until after our planed offline discussion before taking any action on this.

Reply:
added shim function as suggested